control-plane/grpc/proto/v1/replica/replica.proto

```diff
@@ -25,6 +25,8 @@ message Replica{
   string uri = 8;
   // status of the replica
   ReplicaStatus status = 9;
+  // uuid of the pool
+  optional google.protobuf.StringValue pool_uuid = 10;
```

Comment:
Doesn't StringValue already make it optional?

Reply:
Yes, it does but just for readability purpose, I added it as optional. Adding optional here doesn't have any affect though on the generated code.